codespell_lib/data/dictionary.txt

```diff
@@ -14942,8 +15026,38 @@ costructions->constructions
 costructor->constructor
 costum->custom, costume,
 costumary->customary
+costumisable->customisable
+costumisation->customisation
+costumisations->customisations
+costumise->customise
+costumised->customised
+costumiser->customiser
+costumisers->customisers
+costumising->customising
+costumizable->customizable
+costumization->customization
+costumizations->customizations
 costumize->customize
+costumized->customized
+costumizer->customizer
+costumizers->customizers
+costumizes->customizes
+costumizing->customizing
 costums->customs, costumes,
+cosumable->consumable
+cosumables->consumables
+cosume->consume, costume,
+cosumed->consumed
+cosumer->consumer
+cosumers->consumers
+cosumes->consumes, costumes,
+cosuming->consuming
+cosumption->consumption
+cosumptions->consumptions
+cotact->contact
```

Comment:
No `coact` typos?

Reply:
I couldn't find any typos. `coact` is too rare.